test/suite.go

```
@@ -65,18 +73,51 @@
+		s.Run(c.name, func() {
+			err := s.TxInjector.InjectTxs(c.txs)
+			s.Require().NoError(err)
+			txs, _ := s.Exec.GetTxs(ctx)
+			fmt.Println(len(txs))
+			stateRoot, maxBytes, err := s.Exec.ExecuteTxs(ctx, c.txs, uint64(i+1), genesisTime.Add(time.Duration(i)*time.Second), lastStateRoot)
+			s.Require().NoError(err)
+			s.Require().NotEmpty(stateRoot)
+			s.Assert().Equal(c.stateRootChanged, !bytes.Equal(lastStateRoot, stateRoot))
+			s.Require().Greater(maxBytes, uint64(0))
+			lastStateRoot = stateRoot
+			s.Exec.SetFinal(ctx, uint64(i+1))
```

golangci-lint (1.62.2)

Error return value of `s.Exec.SetFinal` is not checked

(errcheck)